vortex-ffi/src/lib.rs

```diff
@@ -18,6 +18,10 @@ pub use log::vx_log_level;
 use tokio::runtime::{Builder, Runtime};
 use vortex::error::VortexExpect;
 
+#[cfg(not(miri))]
+#[global_allocator]
+static GLOBAL: mimalloc::MiMalloc = mimalloc::MiMalloc;
```

Comment:
can't find any definitive answer about how this behaves. Let's go with it for now and watch if anything breaks